<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ihm_UserGuides</class>
 <widget class="QMainWindow" name="ihm_UserGuides">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTextEdit" name="useguides_textedit"/>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="userguides_list"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser_guide_editor">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUserGuideEditor"/>
   </widget>
   <addaction name="menuUser_guide_editor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUserGuideEditor">
   <property name="text">
    <string>User Guide Editor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
